src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.logo__image {
+  width: 100%;
+  height: 100%;
+}
+
+.ul__list {
+  display: flex;
+  height: 60px;
```

Comment:
I don't think that we really need to have this height, we can just set paddings to links